scripts/v.in.wfs/v.in.wfs.py

```diff
@@ -63,6 +63,20 @@
 #% label: Skip earlier feature IDs and start downloading at this one
 #% description: (default: start with the first feature)
 #%end
+#%option
+#% key: user
+#% type: string
+#% required: no
+#% multiple: no
+#% label: Username or file with username or environment variable name with username
```

Comment:
Let me suggest to have consistency with r.in.wms:

https://github.com/OSGeo/grass/blob/27fa95ce8c39e276a4a0201feeed7d0e4fcfb425/scripts/r.in.wms/r.in.wms.py#L110

to avoid confusion.